CMakeLists.txt

```diff
@@ -144,9 +146,31 @@ if( DEFINED DD4HEP_WITH_GEANT4 OR DEFINED DD4HEP_WITH_GEAR )
   dd4hep_print_cmake_options( OPTIONAL "Obsolete options: only the following are allowed:" ERROR 1 )
 endif()
 
+IF(DD4HEP_USE_EXISTING_DD4HEP)
+
+  MESSAGE(STATUS "|!!> Using pre-existing DD4hep: ${DD4HEP_USE_EXISTING_DD4HEP}")
+  SET(CMAKE_PROJECT_NAME DD4hepSelected)
+  dd4hep_set_version(${CMAKE_PROJECT_NAME}
+    MAJOR ${${CMAKE_PROJECT_NAME}_VERSION_MAJOR}
+    MINOR ${${CMAKE_PROJECT_NAME}_VERSION_MINOR}
+    PATCH ${${CMAKE_PROJECT_NAME}_VERSION_PATCH} )
```

Comment:
Do we need a check that if you build for instance DDG4 against an already build DDCore that the versions match? I mean the version of the sources currently being build against the version of the binaries?